### PR TITLE
Task_4-CORE: View a list of all articles

### PR DESCRIPTION
**Pull request by Sailynx-Zenefer**
**Task_4-CORE: View a list of all articles**
- Sets up initial structure of app.
  - Creates Header and Nav components,which will be visible on all pages.
  - Creates main-page component, [ListArticlesView] on '/' which will be a list of all articles.
- Adds data fetching using axios to App.jsx to fetch article data and store it in state.
- Adds [ArticlesLister], a component that generates a list of [ArticleCards] from article data in state.
- Adds [ArticleCard], a component that uses article data in state, formats it and displays it.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,11 +1,36 @@
...
-  const [count, setCount] = useState(0)
+  const [articles, setArticles] = useState([])
+
+  useEffect(() => {
+    axios
+    .get(`${apiAddress}/api/articles`)
+    .then(({data : {articles}}) => setArticles(articles))
+    .catch((err)=> console.log(err))
+    .finally(()=> {})
+  },[])
```
Comment (nc-kirsty-butterworth) [inline src/App.jsx]: this axios request should be in a separate utils file 👀

**Review by nc-kirsty-butterworth — CHANGES_REQUESTED** [review]
really nice work so far 🚀 please extract your api and resubmit. Although you don't need to focus on your styling too much yet your Nav and Header components are taking up a lot of space on the page 👍

**Comment by nc-saleh** [comment]
Looks good Willow! Axios does provide you with an easier way to handle the baseUrl of your api but if you have it working , then I am happy for you to carry on

**Comment by Sailynx-Zenefer** [comment]
> Looks good Willow! Axios does provide you with an easier way to handle the baseUrl of your api but if you have it working , then I am happy for you to carry on

cheers. I'll merge and check that axios thing out then add it on the current commit i'm working on.

**Pull request merged**